org.hlwd.sonofman.yml

```diff
@@ -0,0 +1,16 @@
+app-id: org.hlwd.sonofman
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: som
+finish-args:
+  - --filesystem=home
```

Comment:
this doesn't have a UI?

Reply:
It's a terminal application (console application)